apps/src/templates/codeReviewGroups/codeReviewGroupManagerHooks.js

```diff
@@ -0,0 +1,95 @@
+import {useState} from 'react';
```

Comment:
some style/organization nits for this file:
- the file should be named the same as the hook
- we don't have a standard for this yet, but hooks are generally in their own `hooks/` directory -- making this change may help with the above bullet too, making it clear that this file contains a hook